bin/probot-scaffold.js

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const path = require('path');
+const superb = require('superb');
+const inquirer = require('inquirer');
+const program = require('commander');
+const {scaffold} = require('egad');
+const kebabCase = require('lodash.kebabcase');
+const chalk = require('chalk');
+const stringifyAuthor = require('stringify-author');
+const {guessEmail, guessAuthor, guessUsername} = require('../lib/user');
+
+const PLUGIN_REPO_URL = 'https://github.com/probot/plugin-template.git';
+
+program
+  .usage('[options] [destination]')
+  .option('-p, --pkgname <package-name>', 'Plugin package name')
+  .option('-d, --description "<description>"',
+    'Plugin description (contain in quotes)')
+  .option('-a, --author "<full-name>"',
+    'Plugin author name (contain in quotes)')
+  .option('-e, --email <email>', 'Plugin author email address')
+  .option('-h, --homepage <homepage>', 'Plugin author\'s homepage')
+  .option('-u, --user <username>', 'GitHub username or org (repo owner)')
+  .option('-r, --repo <repo-name>', 'Plugin repo name')
+  .option('--template <template-url>',
+    'URL of alternative plugin template',
+    PLUGIN_REPO_URL)
+  .parse(process.argv);
+
+const destination = program.args.length ?
+  path.resolve(process.cwd(), program.args.shift()) :
+  process.cwd();
+
+const prompts = [
+  {
+    type: 'input',
+    name: 'name',
+    default(answers) {
+      return answers.repo || kebabCase(path.basename(destination));
+    },
+    message: 'Plugin\'s package name:',
+    when: !program.pkgname
+  },
+  {
+    type: 'input',
+    name: 'description',
+    default: `A ${superb()} Probot plugin`,
+    message: 'Description of plugin:',
+    when: !program.description
+  },
```

Comment:
Yeah, I think that's because `commander` has absconded with `program.description`.  `program.name`, which I had tried to use for something else, is a function.  This is all very strange.  I'm not a big fan of commander; there's a bit too much magic for me.

Re `superb`: Yeoman uses it, so I wanted to conform.